tomcat-job-type/job-template.Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM gradle:jdk21 as build
+WORKDIR /src
+COPY . .
+RUN gradle war
+
+#---------------------------
+FROM {{ base_image }}
+
+{% for env_key, env_value in env_vars.items() %}
+ENV {{ env_key }} "{{ env_value }}"
+{% endfor %}
+
+{% if manifest.system_dependencies and manifest.system_dependencies|length > 0 %}
+RUN apk add \
+    {{ manifest.system_dependencies | join(' ') }}
+{% endif %}
+
+RUN sed -i 's/port="8080"/port="7000"/' ${CATALINA_HOME}/conf/server.xml
```

Comment:
Looks like this line can be placed earlier in a base.Dockerfile, since it doesn't depend on a specific job.

Reply:
Good point